core/src/Streamly/Internal/Data/Stream/Exception.hs

```diff
@@ -9,14 +9,7 @@
 module Streamly.Internal.Data.Stream.Exception
     (
       before
-    , after_
```

Comment:
Do not remove anything from this module. We just have to change these to IO in the next PR. It is much better to just make small changes to these to adapt to Io rather than removing these first and then reintroducing again for IO variants.